keel-ec2-api/src/main/kotlin/com/netflix/spinnaker/keel/api/ec2/ServerGroup.kt

```diff
@@ -69,6 +69,7 @@ data class ServerGroup(
   }
 
   data class ActiveServerGroupImage(
+    @get:ExcludedFromDiff
     val imageId: String,
     val appVersion: String?,
     val baseImageVersion: String?,
```

Comment:
There seems to be no compelling reason to include `imageId` in the diff we generate. It's meaningless to users and the information we need to decide whether to redeploy is contained in `appVersion` and `baseImageVersion`. If either of those changes, we'll re-deploy. The `imageId` field is necessary when we launch the Orca task but not in the diff.